packages/phone-number-privacy/signer/package.json

```diff
@@ -18,6 +18,7 @@
     "ssl:keygen": "./scripts/create-ssl-cert.sh"
   },
   "dependencies": {
+    "@celo/base": "1.1.1-dev",
```

Comment:
Will this work with our dockerfile building since its not available?

Reply:
hmm good question. I think this will also come up in Cody's PR

Reply:
Yea I think you're right. One workaround would be to import it in through the common package (which we're already doing in Cody's combiner timeout PR). We'd need to repubiish the package though

Reply:
Just pushed this change